transport/server.hh

```diff
@@ -104,6 +104,7 @@ struct cql_query_state {
 struct cql_server_config {
     ::timeout_config timeout_config;
     size_t max_request_size;
+    uint32_t max_concurrent_requests;
```

Comment:
Here too - using the uint32_t saves us nothing since the next value is going to be 64-bit aligned anyway.